numpy/__init__.pyi

```diff
@@ -2537,11 +2537,22 @@ class object_(generic):
 
 object0 = object_
 
+# The `datetime64` constructors requires an object with the three attributes below, 
+# and thus supports datetime duck typing
+class _DatetimeScalar(Protocol):
+    @property
+    def day(self) -> int: ...
+    @property
+    def month(self) -> int: ...
+    @property
+    def year(self) -> int: ...
+
+
 class datetime64(generic):
-    @overload
+    @overload    
```

Comment:
```suggestion
    @overload
```
Style nit: there are some trailing whitespaces here.

Reply:
Thank you kindly for your patience. I will make a greater effort to include matters of style to my contributions. 

I used a remove trailing white space tool and it found one after `# The `datetime64` constructors requires an object with the three attributes below, `. Did you miss this one or are comments ignored?